core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -54,6 +56,7 @@
  * 
  * @since 2.0
  */
+@Restricted(NoExternalUse.class) // use by Jelly
 public class SetupWizard {
```

Comment:
Probably more of a copy/paste issue. `DoNotUse.class` is the real Jelly use case.